config/initializers/quickbooks.rb

```diff
@@ -0,0 +1,13 @@
+unless Rails.env == 'test'
```

Comment:
I think it would be best if we could pair @linesarefuzzy on a way to clean this up a bit.